packages/aila/src/lib/agentic-system/compatibility/helpers/createTextStreamer.ts

```diff
@@ -0,0 +1,20 @@
+export type TextStreamer = (text: string) => void;
+
+export const createTextStreamer =
```

Comment:
This is fine, but let's also have a stab at a version which doesn't stream by chunk and see if it works ok